src/cascadia/TerminalSettingsEditor/PreviewConnection.cpp

```diff
@@ -16,7 +16,7 @@ static constexpr std::wstring_view PreviewText{
     L"\x1b[36m@@ -1 +1 @@\x1b[m\r\n"
     L"\x1b[31m-    Windows Console\x1b[m\r\n"
     L"\x1b[32m+    Windows Terminal!\x1b[m\r\n"
-    L"C:\\> \x1b[93mWrite-Host \x1b[36m\"\xd83e\xde9f!\"\x1b[1D\x1b[m"
+    L"C:\\> \x1b[93mWrite-Host \x1b[36m\"\xd83c\xdf2f!\"\x1b[1D\x1b[m"

```

Comment:
```suggestion
    L"C:\\> \x1b[93mWrite-Host \x1b[36m\"\U0001F32F!\"\x1b[1D\x1b[m"
```
(I have not tested this.)